pkg/download/classic/download.go

```diff
@@ -100,7 +100,7 @@ func downloadConfigs(client client.ConfigClient, api api.API, projectName string
 	logger := log.WithFields(field.Type(api.ID))
 	foundValues, err := findConfigsToDownload(client, api, filters)
 	if err != nil {
-		logger.WithFields(field.Error(err)).Error("Failed to fetch configs of type '%v', skipping download of this type. Reason: %v", api.ID, err)
+		logger.WithFields(field.Error(err)).Warn("Failed to fetch configs of type '%v', skipping download of this type. Reason: %v", api.ID, err)
 		return results
```

Comment:
```suggestion
		return []downloadedConfig{}
```

Reply:
Same nit for line 110

Reply:
true for this and line 110 there won't ever be anything in `results` - if fine by you, I'd elect to leave this as is for now though, as it's not functionally changing anything and is unrelated to the functional change of this PR.